tests/integration/test_transport_unicode_handling.py

```diff
@@ -0,0 +1,34 @@
+from sinch.domains.verification.models.responses import (
+    StartFlashCallInitiateVerificationResponse,
+    GetVerificationStatusByIdResponse
+)
+
+UNICODE_POTATO = '🥔'
+
+
+def test_sending_unicode_potato_using_verification_api(
+    sinch_client_sync,
+    phone_number
+):
+    verification_response = sinch_client_sync.verification.verifications.start_flash_call(
+        identity={
+            "type": "number",
+            "endpoint": phone_number
+        },
+        reference=UNICODE_POTATO,
+        dial_timeout=5
+    )
+
+    assert isinstance(verification_response, StartFlashCallInitiateVerificationResponse)
```

Comment:
nit
I suppose you can check send and receive in just one function and one mock

Reply:
Ah, also, would be nice to see a ref to the mock in PR description

Reply:
> nit I suppose you can check send and receive in just one function and one mock

Wiremock is doing that behind the scenes :)

Reply:
> Ah, also, would be nice to see a ref to the mock in PR description

👍

Reply:
> Wiremock is doing that behind the scenes :)

Always forgot that 😅